.circleci/config.yml

```diff
@@ -25,7 +25,14 @@ jobs:
                   pip install numpydoc
                   pip install nipype
                   pip install h5py
+                  pip install PyQt5
                   conda install graphviz pydot
+                  git clone https://github.com/EtienneCmb/visbrain.git visbrain/
+                  cd visbrain/
+                  python setup.py develop
+                  git fetch origin feature/mpl_preview
+                  git checkout feature/mpl_preview
+                  cd ..
```

Comment:
yikes ... any chance of using a release version or at least master @annapasca @EtienneCmb ?

Reply:
Yes, it's now on master